.circleci/generate.sh

```diff
@@ -30,13 +30,12 @@ die ()
 
 print_help()
 {
-  echo "Usage: $0 [-l|-m|-h|-f|-e]"
-  echo "   -a Generate the default config.yml using low resources and the three templates"
-  echo "      (config.yml.LOWRES, config.yml.MIDRES and config.yml.HIGHRES). Use this for"
-  echo "      permanent changes in config-2_1.yml that will be committed to the main repo."
-  echo "   -l Generate config.yml using low resources"
-  echo "   -m Generate config.yml using mid resources"
-  echo "   -h Generate config.yml using high resources"
+  echo "Usage: $0 [-f|-p|-a|-e|-i]"
+  echo "   -a Generate the default config.yml using resources compatible with CircleCI's free tier"
```

Comment:
nit: referring to `config.yml` on -a, -f, and -p muddies it up and it's not immediately clear how the relationship works. Noodling on this a bit, maybe something like:
```
-a Generate a new default config.yml to be committed after changes to the template <etc>
-f Generate a config.yml for tests compatible with the CircleCI free tier resources
-f Generate a config.yml for tests compatible with the CircleCI paid tier resources
```

It's subtle, but forklifting the "hey, we're generating a new default config to be committed" up front as well as the slight massaging on -f and -p imply the relationship between these params and the workflow for the config in a way the phrasing here doesn't.

Maybe. Take it or leave it. 😀 I know how it works and had to squint a bit and think hard to make sure I grokked the difference though.

Reply:
Makes sense. I have tried to rephrase using CircleCI terminology, where `config.yml` is the "expanded config" and config-2_1.yml is the "reusable config".